app/scripts/controllers/ticketFinder.ctrl.js

```diff
@@ -60,6 +60,16 @@ angular.module('yourStlCourts').controller('TicketFinderCtrl', function (TicketF
     }
   };
 
+  ctrl.municipalityAdded = function(itemAdded){
```

Comment:
I don't think we should be removing the extra selection but rather make the field invalid and just prevent them from continuing until they have 5 or less. Maybe the 6th one they're adding is one they definitely want to keep and they'd rather remove a different one. So in this case, you're making them do several more steps when just 1 would suffice.